eglot-x.el

```diff
@@ -207,6 +207,15 @@ server detects a problem, this extension makes the beginning of
 the debugging process a tiny bit easier."
   :type 'boolean)
 
+(defcustom eglot-x-enable-inactive-code t
+  "If non-nil, enable inactive code notifications support. This
```

Comment:
The first line should be a complete sentence.  So "This" should go to the next line.